cmd/cx/services.go

```diff
@@ -35,3 +46,25 @@ func runServices(c *cli.Context) error {
 
 	return nil
 }
+
+func runServicesUrl(c *cli.Context) error {
+	app, err := appName(c, ".")
+	if err != nil {
+		return err
+	}
+
+	if len(c.Args()) < 1 {
+		return stdcli.Usage(c)
+	}
+
+	name := c.Args()[0]
+
+	s, err := Rack.ServiceGet(app, name)
+	if err != nil {
+		return err
+	}
+
+	fmt.Println(s.Endpoint)
```

Comment:
I have a commit floating around that makes sure this has `https://` in the URL. That made me wonder if we need further care around protocol and port in service endpoints.